catalog/video/video-mpegts-transformer/src/main/java/org/codice/alliance/transformer/video/MpegTsInputTransformer.java

```diff
@@ -136,11 +146,26 @@ public Metacard transform(InputStream inputStream, final String id)
 
             extractStanag4609Metadata(metacard, fileBackedOutputStream);
 
+            extractMediaEncodings(metacard, fileBackedOutputStream);
+
             return metacard;
         }
 
     }
 
+    private void extractMediaEncodings(Metacard metacard, TemporaryFileBackedOutputStream fbos)
+            throws IOException {
+
+        List<Serializable> serializables = TSStream.from(fbos.asByteSource())
+                .map(PESPacket::getStreamType)
+                .distinct()
+                .map(streamTypeToString)
+                .collect(Collectors.<Serializable>toList());
+
+        metacard.setAttribute(new AttributeImpl(AttributeNameConstants.MEDIA_ENCODING,
```

Comment:
https://github.com/codice/ddf/pull/1033 will make media encoding multivalued.  You should be able to cast it : https://github.com/bdeining/alliance/commit/a06a8c57ad7437b7bc5f932cfbafec22ef273854#diff-86b5374aad9c7b5ae164503abedd3675R677